code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -659,6 +659,9 @@
 	set category = "IC"
 	set src = usr
 
+	return ..()
+
+/mob/living/silicon/robot/execute_mode()
```

Comment:
Is this in the wrong area?
It looks like it should be either replacing the mode() above it or not here at all, since it would be blocking the proc above from doing anything

Reply:
I'm leaving this review because it's an interesting confusion here.
So from what I'm seeing, all verbs are meant to have an execute_[NAME] or run_[NAME] now that is run when the MC handles it. 
Something to document for sure, since it's not immediately obvious.

Reply:
there is a comment on the parent proc that states the use of it, its used if the verb has to be run through the queue